src/GDGUkraine/rest_controller.py

```diff
@@ -110,6 +112,9 @@ def create(self, **kwargs):
         req = cherrypy.request
         orm_session = req.orm_session
```

Comment:
You should move validation here, since we may have `user` field missing. In this case there will be no validator-generated output.